### PR TITLE
[Doc] Add doc about Develop Chat flow

### PR DESCRIPTION
**Pull request by lalala123123**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: docs/how-to-guides/develop-a-flow/develop-chat-flow.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+From this document, you can learn how to develop a chat flow by writing a flow yaml from scratch. You can 
+find additional information about flow yaml schema in [Flow YAML Schema](../../reference/flow-yaml-schema-reference.md).
+
+## Flow input data
+
+The most important elements that differentiate a chat flow from a standard flow are **Chat Input** and **Chat History**. A chat flow can have multiple inputs, but **Chat History** and **Chat Input** are required inputs in chat flow.
+
+- **Chat Input**: Chat input refers to the messages or queries submitted by users to the chatbot. Effectively handling chat input is crucial for a successful conversation, as it involves understanding user intentions, extracting relevant information, and triggering appropriate responses.
+
+- **Chat History**: Chat history is the record of all interactions between the user and the chatbot, including both user inputs and AI-generated outputs. Maintaining chat history is essential for keeping track of the conversation context and ensuring the AI can generate contextually relevant responses. Chat History is a special type of chat flow input, that stores chat messages in a structured format.
```
Comment (wangchao1230) [inline docs/how-to-guides/develop-a-flow/develop-chat-flow.md:16]: we should have some examples showing the format of chat history

Comment (lalala123123) [reply docs/how-to-guides/develop-a-flow/develop-chat-flow.md:16]: Add example to show the format of chat history

File: docs/how-to-guides/develop-a-flow/develop-chat-flow.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+:::{admonition} Experimental feature
+This is an experimental feature, and may change at any time. Learn [more](../faq.md#stable-vs-experimental).
+:::
+
+From this document, you can learn how to develop a chat flow by writing a flow yaml from scratch. You can 
+find additional information about flow yaml schema in [Flow YAML Schema](../../reference/flow-yaml-schema-reference.md).
+
+## Flow input data
+
+The most important elements that differentiate a chat flow from a standard flow are **Chat Input** and **Chat History**. A chat flow can have multiple inputs, but **Chat History** and **Chat Input** are required inputs in chat flow.
```
Comment (crazygao) [inline docs/how-to-guides/develop-a-flow/develop-chat-flow.md:12]: you have already bolded the text, so that you don't need to CAPITALIZE them.

Comment (crazygao) [reply docs/how-to-guides/develop-a-flow/develop-chat-flow.md:12]: same as below, only natural line start needs capitalize.

File: docs/how-to-guides/develop-a-flow/develop-chat-flow.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+- **Chat Input**: Chat input refers to the messages or queries submitted by users to the chatbot. Effectively handling chat input is crucial for a successful conversation, as it involves understanding user intentions, extracting relevant information, and triggering appropriate responses.
```
Comment (crazygao) [inline docs/how-to-guides/develop-a-flow/develop-chat-flow.md:14]: Handling chat input effectively is crucial ...

**Review by wangchao1230 — APPROVED** [review]

File: docs/how-to-guides/develop-a-flow/develop-chat-flow.md (hunk 2 above)
Comment (lalala123123) [reply docs/how-to-guides/develop-a-flow/develop-chat-flow.md]: Changed to lowercase

**Review by crazygao — APPROVED** [review]

**Review by 0mza987 — APPROVED** [review]

**Pull request merged**